### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by sushoff**
new request guided by Alex

### CODE REVIEW

File: skycultures/almagest/constellation_names.eng.fab [hunk 1]
```diff
@@ -1,27 +1,27 @@
 UMi	"Μικρή ʾˊΑρκτος (Mikre Arktos)"	_("Small Bear")
 UMa	"Μεγαλή ʾˊΑρκτος (Megale Arktos)"	_("Great Bear")
-Dra	"Δράκοντος (Drakontos)"		_("Dragon")
+Dra	"Δράκων (Drakun)"		_("Dragon")
 Cep	"Κηφέως (Kepheus)"	_("King Kepheus")
```
Comment (gzotti) [inline skycultures/almagest/constellation_names.eng.fab]: Is it not "-eus" in Greek?

Comment (sushoff) [reply skycultures/almagest/constellation_names.eng.fab]: δράκος = Eye 
δράκων = a dragon (possible form according to Middle Liddell: δρακών, δράκων, sg,masc, nom)

File: skycultures/almagest/constellation_names.eng.fab [hunk 2]
```diff
@@ -30,20 +30,20 @@ Sco	"Σκορπίος (Skorpios)"	_("Scorpion")
...
-Cet	"Κη̃τος (Ketos)"	_("Sea monster")
+Psc	"ʾΙχθύς (Ichthys)"	_("Fish")
+Cet	"Κῆτος (Ketos)"	_("Sea monster")
 Ori	"ʾΩρίων (Orion)"	_("Hero Orion")
-Eri	"Ποταμὸς (Potamos)"	_("River")
-Lep	"Λαγω̃ς (Lagos)"	_("Hare")
+Eri	"Ποταμός (Potamos)"	_("River")
+Lep	"Λαγῶς (Lagos)"	_("Hare")
 CMa	"Κύων (Kyon)"	_("Dog")
 CMi	"Πρόκυνος (Prokynos)"	_("The One before the Dog")
```
Comment (gzotti) [inline skycultures/almagest/constellation_names.eng.fab]: Gen. (??)

File: skycultures/almagest/constellation_names.eng.fab [hunk 3]
```diff
@@ -30,20 +30,20 @@ Sco	"Σκορπίος (Skorpios)"	_("Scorpion")
...
 Arg	"ʾΑργο (Argo)"	_("The ship Argo")
 Hya	"ʽˊΥδρος (Hydros)"	_("Water snake")
-Crt	"Κρατ̀ηρ (Krater)"	_("Vase")
+Crt	"Κρατήρ (Krater)"	_("Vessel")
 Crv	"Κóραξ (Koras)"	_("Raven")
```
Comment (gzotti) [inline skycultures/almagest/constellation_names.eng.fab]: Korax, or are there new phonetic rules?

File: skycultures/almagest/constellation_names.eng.fab [hunk 4]
```diff
@@ -30,20 +30,20 @@ Sco	"Σκορπίος (Skorpios)"	_("Scorpion")
...
 Cen	"Κένταυρος (Kentauros)"	_("Kentaur")
 Lup	"Θηρίον (Therion)"	_("Beast")
 Ara	"Θυμιατήριον (Thymiaterion)"	_("Altar")
 CrA	"Στέφανος Νότιος (Stephanos Notios)"	_("Southern Crown")
-PsA	"Νότιος ʼΙχθύος (Notrios Ichthyos)"	_("Southern Fish")
-Com "πλόκαμος (Plokamos, in Leo)" _("The Lock [of Queen Berenike, in Leo]")
\ No newline at end of file
+PsA	"Νότιος ʼΙχθύς (Notrios Ichthys)"	_("Southern Fish")
```
Comment (gzotti) [inline skycultures/almagest/constellation_names.eng.fab]: Notios (in transcription)

File: skycultures/almagest/description.de.utf8 [hunk 5]
```diff
@@ -8,23 +8,41 @@ Ptolem&auml;us, der &auml;ltere und eigene Beobachtungen in den
...
+leicht berechnen. Für die Genauigkeit von Himmelsgloben ist diese Abschätzung legitim und hinreichend; heute brauchen wir es 
+freilich für viele Anwendungen genauer.  
+</p>
 
 <h2>Wie wurde diese Sky Culture erstellt?</h2></p>
 
+<p> Das Format der Angaben im <i>Almagest</i> folgt einem simplen Schema: "Beschreibung des Sterns in der Figur, Längengrad, 
+Breitengrad, Helligkeit (Bsp.: Ellbogen von X, Länge, Breite, Magnitude). Daher können wir diesen Sternkatalog zur Rekonstruktion 
+der Figurbilder nutzen. Das ist wie "malen nach Zahlen", da Teile der Figur mit Koordinaten angegeben sind. Ein einfaches Beispiel
+führen wir anhand des Sternbildes Rabe vor und listen in der Tabelle darunter auf, wo wir bei anderen Sternbilder ggf. interpolieren
```
Comment (gzotti) [inline skycultures/almagest/description.de.utf8]: ... bei anderen SternbilderN ...

File: skycultures/almagest/description.de.utf8 [hunk 6]
```diff
@@ -8,23 +8,41 @@ Ptolem&auml;us, der &auml;ltere und eigene Beobachtungen in den
...
+mussten oder eine Linie durch nicht gelistete Sterne legen, um die Figur in der Weise erkennbar zu machen, wie Ptolemäus es sich 
+laut seinen Beschreibungen vorgestellt (bzw. auf seinem Himmelsglobus gezeichnet) hatte.  
+</p>
 <p >Der Prozess der Erstellung dieser Sternbilderkultur kann am Beispiel des Sternbilds des Raben gut nachvollzogen werden: </p>
```
Comment (gzotti) [inline skycultures/almagest/description.de.utf8]: Dieser Satz könnte redundant sein.

File: skycultures/almagest/description.en.utf8 [hunk 7]
```diff
@@ -1,34 +1,41 @@
...
 <p >
  The star catalogue consists of
-13 sections, called books.  In the seventh book, Ptolemy listed total
-1022 stars in 48 constellations (among which are about 5 star-like
+13 sections, called books.  In the seventh and eighth book, Ptolemy listed a total
+of 1022 stars in 48 constellations (among which are about 5 star-like
 objects like globular star clusters).  It is a table of coordinates
 of the stars with a description of their position in the
-constellation.  He used ecliptic coordinates.  As one of the first of
+constellation (such as "star at the ellbow of X, longitude, latitude, magnitude").
```
Comment (gzotti) [inline skycultures/almagest/description.en.utf8]: elbow

File: skycultures/almagest/description.en.utf8 [hunk 8]
```diff
@@ -1,34 +1,41 @@
...
-presents the final displacement of the heliocentric model by
-Aristarch and mostly relies on mathematical calculation than on
-physical observations, which explains discrepancies in the
-coordinates written in the  <i>Almagest</i>. </p>
+down to magnitude of 6, for the faintest stars visible.
+</p>
+
+<p>For the computation of planet positions Ptolemy used a
+geocentric model (which is allowed because the observer always stands on Earth) and, thus, 
+contributed to the scientific believe that the heliocentric model by
```
Comment (gzotti) [inline skycultures/almagest/description.en.utf8]: belief

File: skycultures/almagest/description.en.utf8 [hunk 9]
```diff
@@ -1,34 +1,41 @@
...
+Aristarch was not needed. However, for the star catalogue of the <i>Almagest</i> 
+this does not matter at all because a star catalogue always represents a sort of inventory 
+of the fixed dots in the sky and Ptolemy's proclaimed aim of the catalogue is to provide a list
+for making a celestial globe (see Alm., VIII, 3). He uses ecliptical coordinates
+because being aware of precession he wants to ease the business of future globemakers as they 
+might wish to transform his catalogue into coordinates of their epoch. </p>
```
Comment (gzotti) [inline skycultures/almagest/description.en.utf8]: Maybe add (for the non-initiated), or combine the previous sentence with:

Precession as known to Ptolemy shifts all stars parallel to the ecliptic, therefore updates to the epoch of the respective globe maker  could be applied by a simple subtraction of the value for ecliptic longitude.

File: skycultures/almagest/description.en.utf8 [hunk 10]
```diff
@@ -463,10 +467,10 @@ the reconstruction (column 6).
 	<tr valign="top">
 		<td> Crt 		</td>
 		<td>&Kappa;&rho;&alpha;&tau;&#768;&eta;&rho; 		</td>
-		<td> Krater 		</td>
+		<td> Krater (vessel)	</td>
 		<td> Crater 		</td>
 		<td> 		</td>
-		<td> a Krater is special Greek vase: <a href="https://en.wikipedia.org/wiki/Krater">https://en.wikipedia.org/wiki/Krater</a> Ptolemy
+		<td> a Krater is special Greek mixing vessel which looks like an amphora or vase: <a href="https://en.wikipedia.org/wiki/Krater">https://en.wikipedia.org/wiki/Krater</a> Ptolemy
```
Comment (gzotti) [inline skycultures/almagest/description.en.utf8]: explicit: "wine mixing vessel" ?

**Comment by alex-w** [comment]
>As long as this stays in the branch, we can also approve immediately, and Susanne can now I think push to that branch immediately.

After merge patch-1 branch into SH_AlmagestSkyculture and checkout the code from Stellarium/stellarium - yes

File: skycultures/almagest/constellation_names.eng.fab [hunk 11]
```diff
@@ -1,27 +1,27 @@
 UMi	"Μικρή ʾˊΑρκτος (Mikre Arktos)"	_("Small Bear")
 UMa	"Μεγαλή ʾˊΑρκτος (Megale Arktos)"	_("Great Bear")
-Dra	"Δράκοντος (Drakontos)"		_("Dragon")
+Dra	"Δράκων (Drakun)"		_("Dragon")
```
[the remark this answers is not in the document]
Comment (sushoff) [reply skycultures/almagest/constellation_names.eng.fab]: oops, yes, "on" is better  (forgot to change this)
no "\bar{o}" that would be Arabic but not Greek transliteration. I guess, we should leave it without any, I will

File: skycultures/almagest/constellation_names.eng.fab [hunk 12]
```diff
@@ -1,27 +1,27 @@
...
 Boo	"Βοώτης (Bootes)"	_("Ploughman")
 CrB	"Στέφανος (Stephanos)"	_("Crown")
 Her	"ʼΕνγόνασιν (Engonasin)"	_("Kneeler")
-Lyr	"Λύρας (Lyras)"	_("Lyre")
+Lyr	"Λύρα (Lyra)"	_("Lyre")
 Cyg	"ʼˊΟρνιθος (Ornithos)"	_("Bird")
-Cas	"Κασσιεπείας (Kassiepeias)"	_("Queen Kassiepeia")
-Per	"Περσέος (Perseos))"	_("Perseus")
+Cas	"Κασσιέπεiα (Kassiepeia)"	_("Queen Kassiepeia")
+Per	"Περσέως (Perseus))"	_("Perseus")
```
[the remark this answers is not in the document]
Comment (sushoff) [reply skycultures/almagest/constellation_names.eng.fab]: no, this is now correct

File: skycultures/almagest/constellation_names.eng.fab [hunk 13]
```diff
@@ -1,27 +1,27 @@
...
 Aur	"ʾΗνίοχος (Heniochos)"	_("Charioteer")
```
[the remark this answers is not in the document]
Comment (sushoff) [reply skycultures/almagest/constellation_names.eng.fab]: eta?

File: skycultures/almagest/constellation_names.eng.fab [hunk 14]
```diff
@@ -1,27 +1,27 @@
...
+Del	"Δελφίς (Delphis)"	_("Dolphin")
 Equ	"Προτομή (Protomé)"	_("Bust [of a Horse]")
 Peg	"ʻˊΙππος (Hippos)"	_("Horse")
-And	"ʾΑνδρομέδας (Andromeda)"	_("Princess Andromeda")
+And	"ʾΑνδρομέδα (Andromeda)"	_("Princess Andromeda")
 Tri	"Τρίγωνος (Trigonos)"	_("Triangle")
 Ari	"Κριός (Krios)"	_("Ram")
 Tau	"Ταυ̃ρος (Tauros)"	_("Bull")
-Gem	"Δίδυμος (Didimos)"	_("Twins")
+Gem	"Δίδυμοι (Didimoi)"	_("Twins")
```
[the remark this answers is not in the document]
Comment (sushoff) [reply skycultures/almagest/constellation_names.eng.fab]: god, yes! I know, I corrected this - maybe in another file

File: skycultures/almagest/constellation_names.eng.fab (hunk 13 above)
[the remark this answers is not in the document]
Comment (sushoff) [reply skycultures/almagest/constellation_names.eng.fab]: ah, for the Charioteer - yes, definitely

File: skycultures/almagest/constellation_names.eng.fab [hunk 15]
```diff
@@ -1,27 +1,27 @@
...
 UMa	"Μεγαλή ʾˊΑρκτος (Megale Arktos)"	_("Great Bear")
-Dra	"Δράκοντος (Drakontos)"		_("Dragon")
+Dra	"Δράκων (Drakun)"		_("Dragon")
 Cep	"Κηφέως (Kepheus)"	_("King Kepheus")
 Boo	"Βοώτης (Bootes)"	_("Ploughman")
 CrB	"Στέφανος (Stephanos)"	_("Crown")
 Her	"ʼΕνγόνασιν (Engonasin)"	_("Kneeler")
-Lyr	"Λύρας (Lyras)"	_("Lyre")
+Lyr	"Λύρα (Lyra)"	_("Lyre")
 Cyg	"ʼˊΟρνιθος (Ornithos)"	_("Bird")
```
[the remark this answers is not in the document]
Comment (sushoff) [reply skycultures/almagest/constellation_names.eng.fab]: yes! I know, I corrected this - maybe in another file

File: skycultures/almagest/description.de.utf8 (hunk 6 above)
Comment (sushoff) [reply skycultures/almagest/description.de.utf8]: hm, ok, ja: magst du entscheiden, welcher von beiden?

File: skycultures/almagest/description.en.utf8 (hunk 8 above)
Comment (sushoff) [reply skycultures/almagest/description.en.utf8]: y

File: skycultures/almagest/description.en.utf8 (hunk 7 above)
Comment (sushoff) [reply skycultures/almagest/description.en.utf8]: y

File: skycultures/almagest/description.en.utf8 (hunk 10 above)
Comment (sushoff) [reply skycultures/almagest/description.en.utf8]: y

File: skycultures/almagest/constellation_names.eng.fab (hunk 12 above)
[the remark this answers is not in the document]
Comment (gzotti) [reply skycultures/almagest/constellation_names.eng.fab]: My dictionary (Gemoll) says Perseús. Definitely and exclusively with epsilon-ypsilon[withacute]-sigma. The omega-sigma is again the genitive form.

File: skycultures/almagest/description.en.utf8 (hunk 9 above)
Comment (sushoff) [reply skycultures/almagest/description.en.utf8]: ok, please do so

File: skycultures/almagest/description.de.utf8 (hunk 6 above)
Comment (gzotti) [reply skycultures/almagest/description.de.utf8]: Der neue, längere, ersetze den alten (hier: Lösche Zeile 46).

**Pull request merged**